### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by sebas962**
Uploading POM exercises

### CODE REVIEW

File: cypress/integration/buy-shirt.spec.ts [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
+    loginPage.login("aperdomobo@gmail.com", "WorkshopProtractor");
+    // Own locators
+    addressPage.visitAddressCheckOut();
+    shipingStepPage.visitAddShiping();
+    paymentStepPage.visitOrder();
 
-        cy.get("#center_column > div > p > strong")
-            .should("have.text", "Your order on My Store is complete.")
-    });
+    paymentStepPage.verifyQuestion(
```
Comment (holgiosalos) [inline cypress/integration/buy-shirt.spec.ts]: I prefer to call it, verifyConfirmationMessage, and just receive the message we want to validate, without the "have.text".

**Review by holgiosalos — COMMENTED** [review]
As you have included the changes of page objects. You should also add the changes related to the configuration of the github action to run the tests. Also, please remove the package.json and package-lock.json that are under the .github directory

File: cypress/integration/buy-shirt.spec.ts [hunk 2]
```diff
@@ -1,22 +1,33 @@
...
-        cy.get("p > button > span").click()
-        cy.get("#HOOK_PAYMENT > div:nth-child(1) > div > p > a").click()
-        cy.get("#cart_navigation > button > span").click()
+describe("Buy a t-shirt", () => {
+  it("then the t-shirt should be bought", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    productListPage.visitAddProduct();
+    shoppingCartPage.visitMenuShoppingCarte();
+    loginPage.login("aperdomobo@gmail.com", "WorkshopProtractor");
```
Comment (manuelq12) [inline cypress/integration/buy-shirt.spec.ts]: Can you placed email and password as variables please

File: cypress/integration/buy-shirt.spec.ts [hunk 3]
```diff
@@ -1,22 +1,33 @@
...
+    // Own locators
+    addressPage.visitAddressCheckOut();
+    shipingStepPage.visitAddShiping();
+    paymentStepPage.visitOrder();
 
-        cy.get("#center_column > div > p > strong")
-            .should("have.text", "Your order on My Store is complete.")
-    });
+    paymentStepPage.verifyConfirmationMessage("Your order on My Store is complete.");
```
Comment (manuelq12) [inline cypress/integration/buy-shirt.spec.ts]: This should also be a variable

File: package.json [hunk 4]
```diff
@@ -23,13 +26,5 @@
   },
   "dependencies": {
     "cypress": "^9.6.1"
```
Comment (manuelq12) [inline package.json]: Cypress shouldn't be a dependency, only a devDependency, also, did you erase the scripts?

**Review by manuelq12 — CHANGES_REQUESTED** [review]

**Review by manuelq12 — APPROVED** [review]
LGTM

Hey man, if you are going to add more than one step, can you please add them in the description for us to know exactly what to validate? Thank you

**Review by holgiosalos — APPROVED** [review]

**Comment by sebas962** [comment]
> Hey man, if you are going to add more than one step, can you please add them in the description for us to know exactly what to validate? Thank you

I will try to be more precise in my comments

**Pull request merged**